### PR TITLE
Daily Report Sender on the Platform.

### PR DESCRIPTION
**Pull request by caneppelevitor**
image

### CODE REVIEW

File: src/api/notificationsApi.ts [hunk 1]
```diff
@@ -16,10 +17,11 @@ const sendNotification = (subscriberId, payload) => {
         });
 };
 
-const sendEmail = (key) => {
-    return axios
-        .post(`/api/topics/${key}/send`)
+const sendDailyReportEmail = (key, nameSpace, t) => {
+    return request
+        .post(`/topic-subscription/${key}/send/${nameSpace}`)
```
Comment (thesocialdev) [inline src/api/notificationsApi.ts:22]: issue: we should keep the api prefix

File: server/notifications/notification.controller.ts [hunk 2]
```diff
@@ -17,22 +17,15 @@ export class NotificationController {
...
-    ) {
-        const dailyReport =
-            await this.summarizationService.generateDailyReport();
-
-        return this.notificationService.sendDailyReviewsEmail(
-            body.key,
-            dailyReport
+    async sendDailyReport(@Param("key") key, @Param("nameSpace") nameSpace) {
+        const dailyReport = await this.summarizationService.generateDailyReport(
+            nameSpace
```
Comment (thesocialdev) [inline server/notifications/notification.controller.ts:25]: question: what if the nameSpace doesn't exist?

File: src/api/notificationsApi.ts (hunk 1 above)
Comment (thesocialdev) [reply src/api/notificationsApi.ts:22]: never mind, I see now the base URL

File: server/notifications/notification.controller.ts (hunk 2 above)
Comment (caneppelevitor) [reply server/notifications/notification.controller.ts:25]: It will get the Main nameSpace.

**Review by thesocialdev — APPROVED** [review]

**Pull request merged**